### PR TITLE
Twitter

### PR DESCRIPTION
**Pull request by antoncarlsson**

### CODE REVIEW

File: twitter/package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "app.js",
   "scripts": {
+    "start": "node src/app.js"
```
Comment (falapen) [inline twitter/package.json]: Should be ./src/app.js

File: twitter/src/app.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  consumer_secret: twitterConsumerSecret,
+  access_token_key: twitterAccessTokenKey,
+  access_token_secret: twitterAccessTokenSecret
+});
+
+const stream = client.stream('statuses/filter', { track: '', locations: boundingBoxCoordinatesAroundSweden });
+
+stream.on('data', tweet => {
+  if (tweetIsFromASwedishCity(tweet.place)) {
+    const tweet = formatTweet(tweet);
```
Comment (falapen) [inline twitter/src/app.js]: remove const

Comment (falapen) [reply twitter/src/app.js]: Or change variable name

**Review by falapen — CHANGES_REQUESTED** [review]
Change if you want. Nothing major.

File: twitter/package.json (hunk 1 above)
Comment (antoncarlsson) [reply twitter/package.json]: Fixed.

File: twitter/src/app.js (hunk 2 above)
Comment (antoncarlsson) [reply twitter/src/app.js]: Changed the variable name.

**Review by falapen — APPROVED** [review]

**Pull request merged**